test/trento/application/usecases/health_summary_service_test.exs

```diff
@@ -48,49 +49,139 @@ defmodule Trento.HealthSummaryServiceTest do
     end
 
     test "should determine health summary for a SAP System" do
-      %Trento.ClusterReadModel{id: cluster_id} =
+      %ClusterReadModel{id: db_cluster_id} =
         insert(:cluster, type: ClusterType.hana_scale_up(), health: Health.passing())
 
-      %Trento.HostReadModel{id: host_1_id} =
-        host_one = insert(:host, cluster_id: cluster_id, heartbeat: :unknown)
+      %ClusterReadModel{id: app_cluster_id} =
+        insert(:cluster, type: ClusterType.ascs_ers(), health: Health.warning())
 
-      %Trento.SapSystemReadModel{
+      %HostReadModel{id: db_host_id} =
+        db_host = insert(:host, cluster_id: db_cluster_id, heartbeat: Health.unknown())
+
+      %HostReadModel{id: db_host_id_2} =
+        db_host_2 = insert(:host, cluster_id: nil, heartbeat: Health.passing())
+
+      %HostReadModel{id: app_host_id} =
+        app_host = insert(:host, cluster_id: app_cluster_id, heartbeat: Health.passing())
+
+      %HostReadModel{id: app_host_id_2} =
+        app_host_2 = insert(:host, cluster_id: nil, heartbeat: Health.critical())
+
+      %SapSystemReadModel{
         id: sap_system_id,
         sid: sid
       } = insert(:sap_system, health: Health.critical())
 
       insert(:sap_system, deregistered_at: DateTime.utc_now())
 
-      database_instance =
+      database_instances = [
         insert(
-          :database_instance_without_host,
+          :database_instance,
           sap_system_id: sap_system_id,
+          instance_number: "00",
           sid: "HDD",
```

Comment:
Why are these `sid` fields String literals? In our factories file we seem to use `Faker.StarWars.planet()` or `Faker.UUID.v4()` for `sid`s 🤔

Reply:
I'm removing those values from the factory usage. Actually, we don't really care what value they get from the factory.
Anyway, the creation of the `sid` in the factories needs some refactor. We already knew that and I think we might already have some tech-debt ticket to track it